scripts/bert/finetune_classifier.py

```diff
@@ -39,36 +39,24 @@
 import random
 import logging
 import warnings
+import sys
 from functools import partial
 import numpy as np
 import mxnet as mx
 from mxnet import gluon
 from mxnet.contrib.amp import amp
 import gluonnlp as nlp
 from gluonnlp.data import BERTTokenizer
+from gluonnlp.data.classification import tasks
 from gluonnlp.model import BERTClassifier, RoBERTaClassifier
 from gluonnlp.calibration import BertLayerCollector
-from data.classification import MRPCTask, QQPTask, RTETask, STSBTask, SSTTask
-from data.classification import QNLITask, CoLATask, MNLITask, WNLITask, XNLITask
-from data.classification import LCQMCTask, ChnSentiCorpTask
-from data.preprocessing_utils import truncate_seqs_equal, concat_sequences
 
-nlp.utils.check_version('0.9', warning_only=True)
+path = sys.path[0]
+sys.path.append(path + '/../data_utils')
+#pylint: disable=wrong-import-position
```

Comment:
Can this be added to the main API? I got feedbacks that it's not easy to navigate gluonnlp training script because there are multiple helper scripts in data/ model/ and other utils. 

In comparison, gluon-cv training script is always just a single file. 

If the util is model specific, I'm fine with adding model specific API for data ( nlp.data.transform.bert.xxx). It's similar to what's in gluon-cv such as https://gluon-cv.mxnet.io/api/data.transforms.html?highlight=ssd#gluoncv.data.transforms.presets.ssd.SSDDefaultTrainTransform, although i think the API in gluon-cv can be made shorter.

Reply:
Currently we only have BERT and XLNet. The util is suitable for both BERT and XLNet, but I'm not sure if they can satisfy the future models..

Reply:
I'm ok with model specific API - if that's the case, what about can adding .bert or .xlnet before in the API namespace? @leezu wdyt

Reply:
In this case, I think it could be better to have a wrapper for the whole preprocessing(like BERTDatasetTransorm) using these separated functions？

Reply:
Currently I moved the shared parts into main API and leaving the XLNet specific parts under scripts/language.

Reply:
Having model specific API in the main API is fine with me. @zburning as it alleviates the concern of having hard to understand scripts folder, do you see any issue with implementing Haibin's suggestion?

Reply:
Currently, I have data.bert.glue, data.bert.squad, data.xlnet.squad, these three namespaces:
The functions under data.bert.glue are more general(used by both glue&squad finetuning scipts), since it is first designed for bert glue, I placed them under data.bert.glue. 
The functions under data.bert.squad are squad specified, all used by bert squad finetuning and a part used by xlnet squad finetuning.
The functions under data.xlnet.squad are xlnet squad specified, only used by xlnet squad.
Does this address the issue?
@eric-haibin-lin @leezu